### PR TITLE
raw_nnet: Added raw nnet training options in make_configs.py

### PR DESCRIPTION
**Pull request by vimalmanohar**

### CODE REVIEW

File: egs/wsj/s5/steps/nnet3/components.py [hunk 1]
```diff
@@ -180,6 +180,7 @@ def AddFinalLayer(config_lines, input, output_dim,
         use_presoftmax_prior_scale = False,
         prior_scale_file = None,
         include_log_softmax = True,
+        add_final_sigmoid = False,
```
Comment (vijayaditya) [inline egs/wsj/s5/steps/nnet3/components.py]: briefly describe when add_final_sigmoid can be used.

**Comment by vijayaditya** [comment]
I think the preferred way to do this merge is from chain to master. The conflicts arise due to xent-separate-affine option which is not present here. We will have the same issues when master is merged to chain.

**Comment by danpovey** [comment]
the 2 branches are in sync at the moment.

File: egs/wsj/s5/steps/nnet3/tdnn/make_configs.py [hunk 2]
```diff
@@ -449,11 +481,18 @@ def MakeConfigs(config_dir, splice_indexes_string,
     left_context += int(parsed_splice_output['left_context'])
     right_context += int(parsed_splice_output['right_context'])
 
+    add_lda_str = ('true' if add_lda else 'false')
```
Comment (danpovey) [inline egs/wsj/s5/steps/nnet3/tdnn/make_configs.py]: actually, no need for these 2 variables- just use the expressions inline.

File: egs/wsj/s5/steps/nnet3/tdnn/make_configs.py [hunk 3]
```diff
@@ -134,6 +148,9 @@ def CheckArgs(args):
         args.nonlin_input_dim = args.pnorm_input_dim
         args.nonlin_output_dim = args.pnorm_output_dim
 
+    if args.add_final_sigmoid and args.include_log_softmax:
+        raise Exception("--include-log-softmax and --add-final-sigmoid cannot both be true.")
+
     return args
```
Comment (vijayaditya) [inline egs/wsj/s5/steps/nnet3/tdnn/make_configs.py]: if add_final_sigmoid and xent_separate_affine are true raise an exception.

**Comment by danpovey** [comment]
OK,  merging this.

**Pull request merged**